02_configure_host.sh

```diff
@@ -154,18 +154,52 @@ if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
                grep -q BOOTPROTO /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME} || (echo -e "\nBOOTPROTO=dhcp\n" | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-${BAREMETAL_NETWORK_NAME})
            fi
        fi
-        sudo systemctl restart network
+       sudo systemctl restart network
     fi
 fi
 
 # If there were modifications to the /etc/sysconfig/network-scripts/ifcfg-*
 # files, it is required to enable the network service
-if [ "$MANAGE_INT_BRIDGE" == "y" ] || [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
+if [ "$MANAGE_INT_BRIDGE" == "y" ] ; then
```

Comment:
Why is this change needed?

Reply:
I'm sorry. This is a misake. Will revert this change. Must have crept in while testing a few things.